Mathlib/Order/Partition/Equipartition.lean

```diff
@@ -40,6 +40,11 @@ theorem isEquipartition_iff_card_parts_eq_average :
 
 variable {P}
 
+@[simp]
```

Comment:
I'm not sure that this should be `simp` given that `P.IsEquipartition` is already simp-normal?

Reply:
It's often the case that `A` is simp nf, but `¬ A` is not, right? At any rate, I don't really mind so let's remove it for now.
```suggestion
```